src/scss/_sort.scss

```diff
@@ -0,0 +1,48 @@
+/// Tables styles to support sort buttons.
+@mixin oTableSort {
+	// Sort button.
+	.o-table__sort {
+		// sass-lint:disable no-vendor-prefixes
+		-webkit-font-smoothing: antialiased;
+		-moz-osx-font-smoothing: grayscale;
+		-moz-appearance: none;
+		-webkit-appearance: none;
+		// sass-lint:enable no-vendor-prefixes
+		background: none;
+		color: inherit;
+		border: 0;
+		padding: 0;
+		font: inherit;
+		cursor: pointer;
+		user-select: none;
+		padding-right: 20px;
+		min-width: 60px;
+		text-align: left;
+		&:after {
+			@include oIconsGetIcon('arrows-up-down', $container-width: 20,  $container-height: 20, $iconset-version: 1);
+			content: '';
+			margin-right: -20px;
+			vertical-align: middle;
+		}
+	}
+
+	// Show descending icon in sort button with DSC sort applied.
+	[aria-sort].o-table-sorting-descending .o-table__sort,
+	[aria-sort='descending'] .o-table__sort {
+		&:after {
+			@include oIconsGetIcon('arrow-down', $container-width: 20,  $container-height: 20, $iconset-version: 1);
+			vertical-align: middle;
+		}
+	}
+
+	// Show ascending icon in sort button with ascending sort applied or on hover with no sort.
```

Comment:
Maybe a [Matt Hinchliffe maneuvre](https://github.com/Financial-Times/o-expander/blob/6af37b58bed227211362fe1cef603278dc869f15/main.scss#L40)?

Reply:
Haha good shout. Um. My preference is to stick to the correct icon just in case they change and spinning it doesn't work. I confess that feels unlikely.